pkg/util/testing/wrappers.go

```diff
@@ -204,6 +204,16 @@ func (w *WorkloadWrapper) Priority(priority *int32) *WorkloadWrapper {
 	return w
 }
 
+func (w *WorkloadWrapper) PodSets(podSets []kueue.PodSet) *WorkloadWrapper {
+	w.Spec.PodSets = podSets
+	return w
+}
+
+func (w *WorkloadWrapper) PodSetCount(count int32) *WorkloadWrapper {
```

Comment:
This is way too specific. Just use `PodSets` to pass a list with the necessary count.